examples/client/parse_cl.cpp

```diff
@@ -0,0 +1,173 @@
+/******************************************************************************
+**
+** parse_cl.cpp
+**
+** Thu Aug  6 19:42:25 2020
+** Linux 5.4.0-42-generic (#46-Ubuntu SMP Fri Jul 10 00:24:02 UTC 2020) x86_64
+** cerik@Erik-VBox-Ubuntu (Erik Cota-Robles)
+**
+** Copyright (c) 2020 Erik Cota-Robles
+**
+** Definition of command line parser class
+**
+** Automatically created by genparse v0.9.3
+**
+** See http://genparse.sourceforge.net for details and updates
+**
+**
+******************************************************************************/
+
+#include <stdlib.h>
+
+#if defined(_WIN32) || defined(WIN32)
+#include "getopt.c"
```

Comment:
Including this C file from a C++ file won't compile because the C++ compiler will not recognize C-only syntax like the old-style function parameters list. You should put `extern "C"` around the `#include` line.

Also, you might rename the file something like `getopt_impl.h` instead of `getopt.c` since it's included, not compiled.

Reply:
Will be fixed in next iteration.  I am finding Windows build failures painful to address.

Reply:
Yeah Windows is a real pain for development :sweat: